### ISSUE TITLE
Question about some details

### ISSUE THREAD

**Issue by ludonghuan** [abhi4ssj/ReLayNet#4]
Dear Dr. ROY,
I have a few questions about the details of your paper and code:

1) According to the Dataset Section of your paper, you are using the Duke SD-OCT data set and the image size is 512 × 740, but I downloaded the data set and found the size of images are 496x768,.
Have you done some kind of preprocessing?

2) The parameters in  Experimental Settings Section are not the same as in your matlab code, such as batch size, momentum and the change of learning rate is from 10^-1 to 10^-3, then 1-^-4, not by an order of magnitude. Which one should I use to reproduce the result in your paper?

Thanks,
Donghuan Lu

**Comment by abhi4ssj** [comment]
Hi Donghuan,

1. Rows were zero padded to size of 512 (power of 2), for the ease of using encoder-decoder architecture with max-pooling layer, which reduces the resolution by 2. Some of the A-scans at the corners have wrong Ground Truth annotation. This was cropped for training and evaluation purposes. No extra pre-processing was performed.

2. Learning rate was set to 10^-1 to 10^-2, then 10^-3. This was set intuitively. I didn't do a thorough analysis. You may play around with the hyper-parameters for improving it.

Best
Abhijit

**Comment by mmclkv** [comment]
Dear Dr. Roy,
    I also have one question: in this MATLAB codebase you print the confusion matrix of training set and validation set during training, but it seems that you didn't calculate the dice coefficient (the value on the diagonal line is not dice coefficient if I didn't make it wrong), and I can't reproduce the dice coefficient in your paper using your code. Could you please provide more details about the result of dice coefficient in your paper? 

Thanks,
Jiaming Qiu
